framework/PageTable/useTableItems.tsx

```diff
@@ -92,9 +92,19 @@ export interface ISelected<T extends object> {
 
 export function useSelected<T extends object>(
   items: T[],
-  keyFn: (item: T) => string | number
+  keyFn: (item: T) => string | number,
+  defaultSelection?: T[]
```

Comment:
Will this function every time the modal is opened?  If so, I  think  `defaultSelection` is a misnomer and should be changed to something like `selectedItems`, or `value`.

Reply:
This PR does not hook up the default selection for the view into any dialogs.
This just adds the support for the view and future PRs can take advantage of that.